tests/push.bats

```diff
@@ -101,3 +101,46 @@ load helpers
   run_buildah 1 push --signature-policy ${TESTSDIR}/policy.json busybox docker://registry.example.com/evenbusierbox
   expect_output --substring 'push to registry at "registry.example.com" denied by policy: not in allowed registries list'
 }
+
+
+@test "buildah push image to containers-storage" {
+  buildah pull --signature-policy ${TESTSDIR}/policy.json busybox
+  run_buildah push --signature-policy ${TESTSDIR}/policy.json busybox containers-storage:newimage:latest
+  run_buildah images
+  expect_output --substring "newimage"
+  buildah rmi newimage busybox
+}
+
+@test "buildah push image to docker-archive and oci-archive" {
+  buildah pull --signature-policy ${TESTSDIR}/policy.json busybox
+  for dest in docker-archive oci-archive; do
+    mkdir ${TESTDIR}/tmp
+    run_buildah push --signature-policy ${TESTSDIR}/policy.json busybox $dest:${TESTDIR}/tmp/busybox.tar:latest
+    run ls ${TESTDIR}/tmp/busybox.tar
```

Comment:
I don't think you need the 'run' here

Reply:
Thanks for reminding. status check can be covered run the command directly. Will update it.